### PR TITLE
add dftatom package

### PR DESCRIPTION
**Pull request by wolfv**
cc @certik 

Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by wolfv** [comment]
@certik note that this actually only installs the library. not sure if fortran needs something like header files as well?

Btw. you have push rights on my staged-recipes checkout so if you want to change something you can push there.

File: recipes/dftatom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: 7a34311cc2ca5641a2ba81ea4809c972e36ff0e1798498cf6aa9f365ead5ba59
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+    - cmake
```
Comment (xhochy) [inline recipes/dftatom/meta.yaml]: Please also add `make` as a build dependency. It is available by default on x86 images but not on aarch64 or ppc64le.

Comment (wolfv) [reply recipes/dftatom/meta.yaml]: Done!

File: recipes/dftatom/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  license_file: License
+  summary: 'Routines for Radial Integration of Dirac, Schrödinger, and Poisson Equations'
+  description: |
+    This library implements accurate and efficient radial Schrödinger and Dirac shooting-method solvers. 
+    They work with any grid and any numerical potential. In addition, it also implements a radial 
+    Poisson solver and density functional theory self-consistency cycle.
+  doc_url: https://github.com/certik/dftatom/
+
+extra:
+  recipe-maintainers:
```
Comment (xhochy) [inline recipes/dftatom/meta.yaml]: @wolfv Don't you want to be a maintainer?

**Comment by certik** [comment]
Thanks @xhochy for reviewing and thanks @wolfv for packaging it!

**Comment by certik** [comment]
@wolfv it works!

**Comment by wolfv** [comment]
awesome! now @xhochy can merge :)

File: recipes/dftatom/meta.yaml (hunk 2 above)
Comment (wolfv) [reply recipes/dftatom/meta.yaml]: I don't actually know anything about dftatom so ... I think it's fine to just have @certik as maintainer.
Also there are no frequent releases I think :)

File: recipes/dftatom/bld.bat [hunk 3]
```diff
@@ -0,0 +1,17 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      -D CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE ^
```
Comment (wolfv) [inline recipes/dftatom/bld.bat]: @certik you might be interested in including this line in your CMake. I think it also limits the CMake backwards compatibility though.

Comment (certik) [reply recipes/dftatom/bld.bat]: I created an issue for that: https://github.com/certik/dftatom/issues/27.

**Pull request merged**